extensions/westFax/settings.ts

```diff
@@ -0,0 +1,28 @@
+import { type Setting } from '@awell-health/extensions-core'
+import { z, type ZodTypeAny } from 'zod'
+
+export const settings = {
+  username: {
+    key: 'username',
+    label: 'Username',
+    obfuscated: false,
+    required: true,
+    description: 'Your WestFax username',
+  },
+  password: {
+    key: 'password',
+    label: 'Password',
+    obfuscated: true,
+    required: true,
+    description: 'Your WestFax password',
+  },
+} satisfies Record<string, Setting>
+
+export const settingsValidationSchema = z.object({
+  username: z.string().min(1, {
+    message: 'Missing "API URL in the extension settings."',
+  }),
+  password: z.string().min(1, {
+    message: 'Missing "API key in the extension settings."',
```

Comment:
these message make look wrong, why are we talking about API URL and key when validating username and password.